config/assets/default.js

```diff
@@ -1,74 +1,4 @@
 module.exports = {
-  client: {
-    lib: {
-      // Load Angular-UI-Bootstrap module templates for these modules:
-      uibModuleTemplates: [
-        'datepicker',
-        'modal',
-        'popover',
-        'progressbar',
-        'tabs',
-        'tooltip',
-        'typeahead',
-      ],
-      js: [
-        // Non minified versions
-        'node_modules/jquery/dist/jquery.js',
```

Comment:
How does this one get loaded?

Reply:
This is loaded in `config/webpack/webpack.shims.js` which makes it available globally as `jQuery`, `$`, and `window.jQuery`. I will leave this one like this, as it's very fundamental, lots of things expect it in those places.